scripts/jest-setup.ts

```diff
@@ -0,0 +1,39 @@
+import { Client, type ClientConfig } from 'pg';
+import { migrateDb } from '../src/migrator';
+import { getDbConfig } from '../src/test/e2e/helpers/database-config';
+
+let initializationPromise: Promise<void> | null = null;
+const initializeTemplateDb = (db: ClientConfig): Promise<void> => {
+    if (!initializationPromise) {
+        initializationPromise = (async () => {
+            const testDBTemplateName = process.env.TEST_DB_TEMPLATE_NAME;
+            const client = new Client(db);
```

Comment:
I'd extract into resetDB fn so that we have similar level of abstraction inside insitalizeTempalteDb between migrateDb function (encapsulated migrations) and this new code (encapsulated reset db).